src/lib/y2network/widgets/additional_addresses.rb

```diff
@@ -0,0 +1,274 @@
+require "yast"
+require "cwm/custom_widget"
+
+Yast.import "NetworkService"
+Yast.import "LanItems"
+Yast.import "IP"
+Yast.import "Popup"
+Yast.import "String"
+
+module Y2Network
+  module Widgets
+    class AdditionalAddresses < CWM::CustomWidget
+      def initialize(settings)
+        textdomain "network"
+        @settings = settings
+      end
+
+      def contents
+        Frame(
+          Id(:additional_addresses),
+          # Frame label
+          _("Additional Addresses"),
+          HBox(
+            HSpacing(3),
+            VBox(
+              # :-) this is a small trick to make ncurses in 80x25 happy :-)
+              # it rounds spacing up or down to the nearest integer, 0.5 -> 1, 0.49 -> 0
+              VSpacing(0.49),
+              Table(
+                Id(:address_table),
+                Opt(:notify),
+                Header(
+                  # Table header label
+                  _("Address Label"),
+                  # Table header label
+                  _("IP Address"),
+                  # Table header label
+                  _("Netmask")
+                ),
+                []
+              ),
+              Left(
+                HBox(
+                  # PushButton label
```

Comment:
np: I would write `PushButton labels` once.

Reply:
It sadly won't work (I think). I think it need to be before each translation to be properly extracted to potfile.